examples/clean-architecture/routes/notes.route.js

```diff
@@ -0,0 +1,25 @@
+var path = require("path")
+var router = require('../../..').Router();
+
+//Database connection (you can change it to use a different database)
+var NotesRepository = require(path.join(__dirname, '..', 'repositories', 'note.repository'));
```

Comment:
there is no need to create an absolute path to `require`, it's always relative to the file.  this should just be `require('../repositories/note.repository')` for example.